tdd_buckets.h

```diff
@@ -0,0 +1,38 @@
+#include "iostream"
+#include "assert.h"
+#include <bits/stdc++.h>
+#include <vector>
+using namespace std;
+
+vector<int> values = {3, 3, 4, 5, 10, 11, 12};
+
+int count1 = 0;
+bool flag = false;
+
+void checkAndCountUp(){
+    if (flag){
+        count1++;
+    }
+}
+void checkAndStopCount(int value,int &b){

```

Comment:
even to find the values within range, this code seems confusing. We can use simple linear search to check if value is in range.

There is room for improvement from readability and maintainability point of view.

Reply:
@prithamvshetty 
Yes, I have now changed most of the Code structure. Please check now.

Reply:
👍